portal-impl/src/com/liferay/portal/service/impl/PortletLocalServiceImpl.java

```diff
@@ -499,6 +504,28 @@ public List<Portlet> getFriendlyURLMapperPortlets() {
 		return portlets;
 	}
 
+	private long _inferCompanyId(String todoMessage) {
```

Comment:
todoMessage not used?

Reply:
Yes. It was just a trick to remember what needs to be done still. Instead of a comment I used a parameter.